dev/TabView/TabView_themeresources.xaml

```diff
@@ -60,15 +60,7 @@
 
             <StaticResource x:Key="TabViewBorderBrush"                                      ResourceKey="CardStrokeColorDefault" />
             <StaticResource x:Key="TabViewItemBorderBrush"                                  ResourceKey="SubtleFillColorTransparentBrush" />
-            <LinearGradientBrush x:Key="TabViewSelectedItemBorderBrush" MappingMode="Absolute" StartPoint="0,0" EndPoint="0,4">
-                <LinearGradientBrush.RelativeTransform>
-                    <ScaleTransform ScaleY="-1" CenterY="0.5"/>
-                </LinearGradientBrush.RelativeTransform>
-                <LinearGradientBrush.GradientStops>
-                    <GradientStop Offset="1.0" Color="Transparent"/>
-                    <GradientStop Offset="1.0" Color="{ThemeResource CardStrokeColorDefault}"/>
-                </LinearGradientBrush.GradientStops>
-            </LinearGradientBrush>
+            <StaticResource x:Key="TabViewSelectedItemBorderBrush"                          ResourceKey="CardStrokeColorDefault" />
```

Comment:
What is the reason for that change? I don't think that changing this resource makes sense in this PR.

Reply:
![image](https://user-images.githubusercontent.com/84711285/120017084-1af02700-bfee-11eb-83fd-f5845844225f.png)
This and the other change were done because the corners were rendered behind the tab before.
The tab container used a gradient that became transparent where the corners would be, but now you can set `TabViewSelectedItemBorderBrush` and affect the entire tab.

Reply:
This way you beat the purpose of having the "arcs". The gradient exists to make smooth transition from the TabViewItem to the arcs on the sides. If you use SolidColorBrush instead of gradient, you will end up with 
![image](https://user-images.githubusercontent.com/21356912/120044357-89b78b00-bfc2-11eb-8098-03779465ded0.png)

it just beats the purpose of having smooth arcs on the sides:)

Reply:
That image was meant to show what happens if the arcs are left before the tab container. This is how it works with my change:
![image](https://user-images.githubusercontent.com/84711285/120046714-7389e900-c01b-11eb-9ac5-273f911e375c.png)

You can see a bit of the tab border at the bottom because of #5099, but if you used a single color gradient it would look normal.
![image](https://user-images.githubusercontent.com/84711285/120046868-cf547200-c01b-11eb-994c-20c27a68a914.png)

Reply:
this will only work if the background is solid of both the arcs and the and the tabviewitem :)
if they're translucent, you'll see that little piece of tab border in the bottom anyway.
I see what you mean, this might be a good idea, we're using solid colors anyway
And this way we don't have to use the gradient I suppose?

Reply:
Yes, removing the gradient was so that you could use a single brush for both the arcs and the container. But I agree that it's not a good option with a semi-transparent tab.